### PR TITLE
Node.js bindings: Add examples for AIO and PWM JS bindings

### PR DESCRIPTION
**Pull request by nagineni**
This adds the examples for AIO and PWM JS bindings.
- aio_read.js: Read the value of analog device(e.g. Light sensor)  in every one second.
- pwm_led.js: Using PWM, change the brightness of an LED in every 300ms.

Signed-off-by: Sudarsana Nagineni <sudarsana.nagineni@intel.com>

### CODE REVIEW

**Comment by nagineni** [comment]
@edersondisouza @gabrielschulhof Please take a look.

**Comment by gabrielschulhof** [comment]
:+1: though I haven't tested it.

File: src/samples/nodejs/pwm_led.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+ */
+
+/*
+ * Sample code to demonstrate the usage of PWM JS bindings for
+ * configuring and using PWM pins.
+ *
+ * This example changes the brightness of an LED in a loop.
+ *
+ * Pin configuration:
+ * LED (e.g. Grove LED) -> PWM pin
```
Comment (edersondisouza) [inline src/samples/nodejs/pwm_led.js]: Which pin? Which board?

Comment (nagineni) [inline src/samples/nodejs/pwm_led.js]: PWM pin 3 on Edison (D3 on the Grove base shield)

I will update the comment.

**Comment by nagineni** [comment]
Replaced by #1953

**Pull request closed without merging**